pkg/sql/parser/sql.y

```diff
@@ -16456,36 +16456,568 @@ unreserved_keyword:
 
 // Column label --- keywords that can be column label that doesn't use "AS"
 // before it. This is to guarantee that any new keyword won't break user
-// query like "SELECT col label FROM table" where "label" is a new keyword.
+// query like "SELECT col label FROM tab" where "label" is a new keyword.
 // Any new keyword should be added to this list.
 bare_label_keywords:
-  AS_JSON
+  ABORT
```

Comment:
It make sense that we want to match psql's grammar, but I don't think we have to, because at this moment user won't have valid `SELECT 1 xxx` queries  against these keywords. I guess one counter point to my opinion is that users upgrading from much older version might run into problems because some keywords added before we have this list heh.
That's being said I'm ok with this.

Reply:
i think it's still worth it to match PG's grammar as close as we can. the reason is that we have potential future users who will be migrating their apps from PG to CRDB. for all these users, we want to make sure that queries that worked with PG also work with CRDB, to minimize the friction of migrating to CRDB.